### PR TITLE
Add JVS recipe

### PR DESCRIPTION
**Pull request by kan-bayashi**
This PR add JVS TTS recipe.
(Should be merged after #1257)

- [x] make recipes `egs/jvs/tts1`
- [x] update `egs/REAMDE.md`
- [x] add `egs/jvs/README.md`
- [x] upload pretrained model and add `egs/jvs/tts1/RESULTS.md`

Additionally I made some changes in `egs/jsut/tts1`
- update `data_prep.sh` [commit](https://github.com/espnet/espnet/pull/1258/commits/00f0b127e31fd73b5428079fe3eabafdc53d0248)
- changed basename of data directory. [commit](https://github.com/espnet/espnet/pull/1258/commits/c83d31e051b2c059f1b65848c70086c2ba846b76)
- fixed `clean_text.py` [commit](https://github.com/espnet/espnet/pull/1258/commits/f31ef96283054caa361e0d78292a8711c7a2e239)

### CODE REVIEW

**Comment by kan-bayashi** [comment]
Almost done.

File: egs/jsut/tts1/run.sh [hunk 1]
```diff
@@ -141,7 +141,7 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
 fi
 
 if [ -z ${tag} ]; then
-    expname=${train_set}_${backend}_$(basename ${train_config%.*})_${trans_type}
+    expname=${train_set}_${backend}_$(basename ${train_config%.*})
```
Comment (r9y9) [inline egs/jsut/tts1/run.sh]: Is it intentional? I thought that adding trans_type here is convenient.

Comment (kan-bayashi) [reply egs/jsut/tts1/run.sh]: I added `${trans_type}` to the head of `${train_set}`.
See this commit.
https://github.com/espnet/espnet/pull/1258/commits/c83d31e051b2c059f1b65848c70086c2ba846b76

Comment (r9y9) [reply egs/jsut/tts1/run.sh]: Oh I see. Thanks!

**Comment by kan-bayashi** [comment]
Finished the initial experiment.
Samples https://drive.google.com/open?id=1jvASMlDrQ8s2mQTRqL-Ru5sELMMYaGNp
Even if we use only 100 utterances for fine-tuning, it works well.

**Comment by kan-bayashi** [comment]
Now ready to merge.

**Comment by kan-bayashi** [comment]
@r9y9 Could you review it? 
I want to merge.

File: egs/jvs/tts1/RESULTS.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# v.0.5.3 / Initial results of fine-tuning
+
+- Initial model: `jsut.24k.phn.transformer`
+- Target speaker: `jvs010`
+- FTT in points: 2048
```
Comment (r9y9) [inline egs/jvs/tts1/RESULTS.md]: typo? you probably meant FFT

File: egs/jvs/tts1/RESULTS.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- Shift in points: 300
+- Window length in points: 1024
```
Comment (r9y9) [inline egs/jvs/tts1/RESULTS.md]: 1200

File: egs/jvs/tts1/RESULTS.md (hunk 2 above)
Comment (kan-bayashi) [reply egs/jvs/tts1/RESULTS.md]: Thanks!

File: egs/jvs/tts1/RESULTS.md (hunk 3 above)
Comment (kan-bayashi) [reply egs/jvs/tts1/RESULTS.md]: Thanks!

**Pull request merged**